### PR TITLE
Added ADAL test sources to its build.gradle file.

### PR DESCRIPTION
**Pull request by HSAR**

### CODE REVIEW

**Comment by heidijinxujia** [comment]
Hi @HSAR are you going to merge this PR? I think it will do the auto-merge and you do not need to rebase it, right? BTW, the status check shows pending "VSTS: ADAL Android CI — Waiting for status to be reported" in other PRs. Is there anything changed in the configure, 'cause it works well previously.

**Comment by HSAR** [comment]
Hey, @heidijinxujia. I need a review, as you see. I don't see anything wrong with the checks - the three regular CIs succeed, while the "CI (Development)" is failed because at the time there were failing tests.

**Comment by heidijinxujia** [comment]
Are you waiting for Nazuk's review? 'cause you dismissed my previous review.

**Comment by HSAR** [comment]
It dismissed because I rebased.

File: adal/build.gradle [hunk 1]
```diff
@@ -46,6 +46,8 @@ android {
             manifest.srcFile 'src/main/AndroidManifest.xml'
             java.srcDirs = ['src/main/java', 'src/main/aidl']
         }
+
+        androidTest.setRoot('src/androidTest')
```
Comment (jaredsburrows) [inline adal/build.gradle]: Can't you just use normal android source sets instead of the eclipse ones?

Comment (HSAR) [inline adal/build.gradle]: This is where the tests are, I'm just including them to enable automated CI scenarios.

Comment (jaredsburrows) [inline adal/build.gradle]: I know. With the current Eclipse based environment setup, you have to manually specify the source sets.

If you use the maven/intellij way, you wouldnt have to do this.

**Pull request merged**